react-accordion-component/my-app/src/Accordion.tsx

```diff
@@ -0,0 +1,47 @@
+import { useState } from 'react';
+import './Accordion.css';
+
+type topics = {
+  topic: string;
+  id: number;
+  information: string;
+};
+
+type Props = {
+  topicArray: topics[];
+};
+
+export default function Accordion({ topicArray }: Props) {
+  return (
+    <div>
+      <Topics topicArray={topicArray} />
```

Comment:
The Topics component, as you have it defined, isn't doing much and doesn't need to be its own component.